src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import duke.tasks.*;
+import duke.command.*;
+import duke.exceptions.*;
```

Comment:
Remember to expand your imports
https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements